gulpfile.js

```diff
@@ -26,6 +31,17 @@ gulp.task(
   gulp.parallel('clean:vsix', () => del(['server', 'out', 'out-cov']))
 );
 
+gulp.task('cycloneDx',function (cb) {
+  const packageJSON = getPackageJSON();
+  const version = packageJSON.version;
+  exec(`npm run cyclonedx-run -- -d --output sonarlint-vscode-${version}.sbom-cyclonedx.json`, function (err, stdout, stderr) {
```

Comment:
Is it on purpose to include dev dependencies?

Reply:
yes